x/stakeibc/keeper/lsm_token_deposit_test.go

```diff
@@ -0,0 +1,64 @@
+package keeper_test
+
+import (
+	"math/rand"
+	"strconv"
+	"testing"
+
+	sdkmath "cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+
+	keepertest "github.com/Stride-Labs/stride/v8/testutil/keeper"
+	"github.com/Stride-Labs/stride/v8/testutil/nullify"
+	"github.com/Stride-Labs/stride/v8/x/stakeibc/keeper"
+	"github.com/Stride-Labs/stride/v8/x/stakeibc/types"
+)
+
+func createSetNLSMTokenDeposit(keeper *keeper.Keeper, ctx sdk.Context, n int) []types.LSMTokenDeposit {
+	items := make([]types.LSMTokenDeposit, n)
+	for i := range items {
+		validatorAddr := "validatorAddress"
+		tokenRecordId := rand.Int()
+
+		items[i].Denom = validatorAddr + strconv.Itoa(tokenRecordId)
+		items[i].ValidatorAddress = validatorAddr
+		items[i].ChainId = strconv.Itoa(i)
+		items[i].Amount = sdkmath.ZeroInt()
+		items[i].Status = types.TRANSFER_IN_PROGRESS
+		keeper.SetLSMTokenDeposit(ctx, items[i])
+	}
+	return items
+}
+
+func TestLSMTokenDepositGet(t *testing.T) {
+	keeper, ctx := keepertest.StakeibcKeeper(t)
+	items := createSetNLSMTokenDeposit(keeper, ctx, 10)
```

Comment:
superrrr nit here and no need to change in this PR since these tests are so straightforward, but the `items/got` naming is some of the generic ignite variable naming.
In future PRs (especially with more complex tests), I would change it to `expected/actual` like:

```go
expectedDeposits := createSetNLSMTokenDeposit(keeper, ctx, 10)
for _, expectedDeposit := range expectedDeposits {
   actualDeposit, found := keeper.GetLSMTokenDeposit(ctx, expectedDeposit.ChainId, expectedDeposit.Denom)
   ...
   s.Require().Equal(expectedDeposit, actualDeposit)
}
```